Automattic/sensei
# Update course navigation to move between both lessons and modules

Fixes #1772.

Currently, when viewing a module, it looks like this:

![screen shot 2018-01-22 at 4 31 34 pm](https://user-images.githubusercontent.com/1190420/35245557-bef43c54-ff91-11e7-94a2-fe7272facb10.jpg)

That is, it shows the module title and description, and then lists all of the lessons that are associated with that particular module. The navigation links at the bottom take the user to the next or previous module.

## New Module Page
This PR changes the module page by removing the list of lessons:

![screen shot 2018-01-22 at 4 33 56 pm](https://user-images.githubusercontent.com/1190420/35245671-116cc6a4-ff92-11e7-93ba-f082c4a230f9.jpg)

The navigation links are also updated such that the user can navigate between both modules and lessons as per the order set in _Lessons_ > _Order Lessons_ in the admin.

## Testing

There are a number of things that should be tested:

1. Check that the module page shows correctly.
2. Check that the course navigation links take the user to the next / previous lesson or module as set in _Lessons_ > _Order Lessons_ of the admin.
3. Check that the navigation links for a quiz are the same as that of the associated lesson (i.e. for a lesson with a quiz, click the _View the Lesson Quiz_ button, and then ensure that the navigation links are the same as those of the lesson.)
4. For a completed lesson, check that the _Next Lesson_ button goes to the next module / lesson of the course:
![screen shot 2018-01-22 at 4 42 17 pm](https://user-images.githubusercontent.com/1190420/35246049-5385c30a-ff93-11e7-803c-58d12131b70b.jpg)
